WapTenantPublicAPI.psm1

```diff
@@ -549,6 +549,192 @@ function GetWAPSubscriptionQuota {
     
 }
 
+function Get-WAPSubscriptionQuota{
+    <#
+    .SYNOPSIS
+        Retrieves a VM Role Quota information object (WAP.Quota).
+
+    .PARAMETER Subscription
+        The Subscription(s) you wish to collect quota information for.
+
+    .EXAMPLE
+        PS C:\>$URL = 'https://publictenantapi.mydomain.com'
+        PS C:\>$creds = Get-Credential
+        PS C:\>Get-WAPToken -Credential $creds -URL 'https://sts.adfs.com' -ADFS
+        PS C:\>Connect-WAPAPI -URL $URL
+        PS C:\>$Sub = Get-WAPSubscription -Name MySubscription 
+        PS C:\>Get-WAPSubscriptionQuota -Subscription $Sub
+    #>
+[CmdletBinding()]
+Param(
+    [Parameter(
+        Mandatory=$False,
+        ValueFromPipeline=$True,
```

Comment:
Lowercase t on $True and there should be spaces surrounding '='  (everywhere ;) )